src/chocolatey/infrastructure.app/services/ChocolateyConfigSettingsService.cs

```diff
@@ -231,6 +231,28 @@ public void ListFeatures(ChocolateyConfiguration configuration)
             }
         }
 
+        public void GetFeature(ChocolateyConfiguration configuration)
+        {
+            var feature = GetFeatureValue(configuration.FeatureCommand.Name);
+            if (feature == null)
+            {
+                throw new ApplicationException("No feature value by the name '{0}'".FormatWith(configuration.FeatureCommand.Name));
+            }
+
+            this.Log().Info("{0}".FormatWith(feature.Enabled ? "Enabled" : "Disabled"));

```

Comment:
This feels a bit wrong IMO.

To follow the convention when we are running against `List`, we should probably include the name of the feature as well.

But if the user specifies limited output, then only `Enabled`/`Disabled` is shown.